entrypoint.sh

```diff
@@ -2,48 +2,75 @@
 set -e
 echo "Configuring..."
 
+COLLECTORS=`ls /opt/netuitive-agent/conf/collectors/ | sed 's%\.conf%%;s%Collector%%'`
+
 if [[ ! $USE_LOCAL_CONFIG ]]; then
 
 	sed -i -e "s/api_key\ =\ apikey/api_key = ${APIKEY}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring APIKEY with $APIKEY"
+	echo "Configuring APIKEY: $APIKEY"
 
 	sed -i -e "s/https/${HTTPVAR}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring HTTPVAR with $HTTPVAR"
+	echo "Configuring HTTPVAR: $HTTPVAR"
 
 	sed -i -e "s/level\ =\ INFO/level\ =\ ${LOGLEVEL}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring loglevel with $LOGLEVEL"
+	echo "Configuring loglevel: $LOGLEVEL"
 
 	sed -i -e "s/interval\ =\ 60/interval\ =\ ${INTERVAL}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring interval with $INTERVAL"
+	echo "Configuring interval: $INTERVAL"
 
 	sed -i -e "s/api\.app\.netuitive\.com/${APIHOST}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring APIHOST with $APIHOST"
+	echo "Configuring APIHOST: $APIHOST"
 
 	sed -i -e "s/# hostname\ =\ my_custom_hostname/hostname\ =\ ${DOCKER_HOSTNAME}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring HOSTNAME with $DOCKER_HOSTNAME"
+	echo "Configuring HOSTNAME: $DOCKER_HOSTNAME"
 
 	sed -i -e "s/listen_ip\ =\ 0.0.0.0/listen_ip\ =\ ${LIP}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring LISTEN_IP with $LIP"
+	echo "Configuring LISTEN_IP: $LIP"
 
 	sed -i -e "s/listen_port\ =\ 8125/listen_port\ =\ ${LPRT}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring LISTEN_PORT with $LPRT"
+	echo "Configuring LISTEN_PORT: $LPRT"
 
 	sed -i -e "s/forward_ip\ =\ 0.0.0.0/forward_ip\ =\ ${FIP}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring FORWARD_IP with $FIP"
+	echo "Configuring FORWARD_IP: $FIP"
 
 	sed -i -e "s/forward_port\ =\ 8125/forward_port\ =\ ${FPRT}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring FORWARD_PORT with $FPRT"
+	echo "Configuring FORWARD_PORT: $FPRT"
 
 	sed -i -e "s/forward\ =\ False/forward\ =\ ${FORWARD}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
-	echo "Configuring FORWARD with $FORWARD"
+	echo "Configuring FORWARD: $FORWARD"
+
+	if [ "${APIURL}" ]; then
+		sed -i -e "s%url =.*%url = ${APIURL}%g" /opt/netuitive-agent/conf/netuitive-agent.conf
+		echo "Configuring URL: $APIURL"
+	fi
+
+	if [ ! "${STATSD_HOST}" ]; then
+		STATSD_HOST="localhost"
+	fi
+	sed -i -e "s/sensu\.app\.netuitive\.com/${STATSD_HOST}/g" /opt/netuitive-agent/conf/netuitive-agent.conf
```

Comment:
This doesn't seem like it should be in the public repo.

Reply:
Good catch!